### PR TITLE
Display existing concept map

### LINKED ISSUE #398: Concept Maps: visualize/modify an existing concept map + select an existing concept map
## Description

~~1. Clicking on a mapped concept map on pyrog should show the existing concept map (mapping between source terminology to FHIR terminology). Now, it just show nothing~~
2. It must be possible to use an existing concept map, it is not for now

## Expectation

1. Show list of available ValueSet in the API (in source and target select) (remove CodeSystem)
2. If a concept map already exists on the two selected value sets, show it (and make it modifiable)
3. (Optional) show the binding strength of the ValueSet precognized by FHIR (ElementDefinition.Binding.Strength) for the selected attribute

See issue https://github.com/arkhn/pyrog/issues/424 to understand how to retrieve values of a ValueSet

## Screenshots

image

## Environment

Staging, Demo

### PR DESCRIPTION
**Pull request by tevariou**
## Fixes

Fixes #398 by @nriss 

## Description

* Display current concept map 

## Technical details

* Pass current concept map to ConceptMapDialog component

## Tests

* Click on concept map name, it should display the concept map

### CODE REVIEW

File: client/src/components/sources/index.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ const SourcesView = (): React.ReactElement => {
         <div id="software-cards">
           {loadingSources ? (
             <Spinner />
-          ) : (
+          ) : dataSources?.sources ? (
```
Comment (Jasopaum) [inline client/src/components/sources/index.tsx:108]: I find it clearer to write `dataSources?.sources ? && <Component>` here

**Review by Jasopaum — APPROVED** [review]
You just have a React hook dependency error but otherwise, LGTM 👍

**Comment by nriss** [comment]
Great job !
The existing concept maps are well displayed
However, when I add a new attribute, I have an empty concept map which make the preview impossible, and impossible to remove it
image

**Comment by nriss** [comment]
It seems that the CodeSystems are not pushed on the API
This query doesn't return any resource : api/CodeSystem?title:contains=Mimic_gender2 on staging

see [ConceptMap post function](https://github.com/arkhn/pyrog/blob/d2413b13835a608717a84089af6d1531df71e5dd/client/src/components/mapping/ConceptMap/index.tsx#L492)

**Pull request merged**